examples/js_library/two/BUILD.bazel

```diff
@@ -32,11 +31,15 @@ bzl_library(
     ],
 )
 
-stardoc_with_diff_test(
-    name = "tsc-docs",
-    bzl_library_target = ":tsc",
-)
-
-update_docs(
-    name = "docs",
-)
+# TODO(derek): stardoc doesn't work under bzlmod with these
+# targets when working against an overridden version of bazel-lib.
+# Remove comments of the released version of bazel-lib succeeds.
+#
+# stardoc_with_diff_test(
+#     name = "tsc-docs",
+#     bzl_library_target = ":tsc",
+# )
+#
+# update_docs(
+#     name = "docs",
+# )
```

Comment:
This can actually go into the second commit which I will remove. The issue only surfaced when using overridden versions which I was doing for testing.

Reply:
Nope, this still needs to be commented out.

Reply:
Can you file an issue so we don't forget to fix it? The generated doc will start drifting out of date

Reply:
Yeah I'll post an issue.

Reply:
https://github.com/aspect-build/rules_js/issues/1362